DB/src/main/java/io/deephaven/db/v2/remote/ConstructSnapshot.java

```diff
@@ -1212,11 +1247,11 @@ public static boolean serializeAllTable(final boolean usePrev,
                                             final Object logIdentityObject,
                                             final BitSet columnsToSerialize,
                                             final Index positionsToSnapshot) {
+        final BitSet addColumns = columnsToSerialize == null ? new BitSet() : (BitSet) columnsToSerialize.clone();
```

Comment:
Maybe the reason for your other exception? The null case should be all, not none.